addon/instance-initializers/firebase-settings.ts

```diff
@@ -15,6 +16,7 @@ interface AuthAddonConfig {
   emulator?: {
     hostname: string,
     port: number,
+    options?: { disableWarnings: boolean },
```

Comment:
We can do `options?: { [key: string]: unknown }` just like in `FirestoreAddonConfig` above so we're flexible for future updates of Firebase as I don't want to release update to the interface per every new experimental config that they add.

Reply:
Good call.

Annoyingly, `disableWarnings` is a *required* property on the optional options object, so I get this TS error with that code:

```
No overload matches this call.
  Overload 1 of 2, '(auth: FirebaseAuth, url: string, options?: { disableWarnings: boolean; } | undefined): void', gave the following error.
    Argument of type 'Auth' is not assignable to parameter of type 'FirebaseAuth'.
      Type 'Auth' is missing the following properties from type 'FirebaseAuth': applyActionCode, checkActionCode, confirmPasswordReset, createUserWithEmailAndPassword, and 16 more.
  Overload 2 of 2, '(auth: Auth, url: string, options?: { disableWarnings: boolean; } | undefined): void', gave the following error.
    Argument of type '{ [key: string]: boolean; } | undefined' is not assignable to parameter of type '{ disableWarnings: boolean; } | undefined'.
      Property 'disableWarnings' is missing in type '{ [key: string]: boolean; }' but required in type '{ disableWarnings: boolean; }'.ts(2769)
```

Should I just include both lines?

Reply:
Yeah, I think that would be better.